requirements.txt

```diff
@@ -8,5 +8,6 @@ mypy
 pytest
 pytest-cov
 requests
+SQLAlchemy==1.1.9
```

Comment:
Let's drop the version here for now. If we start pinning requirements then we should do that throughout, rather than just for this one.

At some point we'll probably also need to start breaking this down into eg `# core requirements`, `# optional packages`, `# test requirements`, with SQLAlchemy being an optional. But let's not think about that for now.